src/lib.rs

```diff
@@ -553,8 +526,47 @@ impl JubJubAffine {
                     BlsScalar::conditional_select(&x, &x_negated, flip_sign);
 
                 CtOption::new(JubJubAffine { x: final_x, y }, Choice::from(1u8))
-            })
-        })
+            }),
+        )
+        .ok_or(BytesError::InvalidData)
+    }
+}
+
+impl DeserializableSlice<32> for JubJubAffine {}
```

Comment:
Remove this, is not necessary – it would actually result in a compilation error now.

Use `cargo update` locally to update your deps, so you can see that it will fails.

Reply:
Apparently I forgot to commit that 😅 thought I already did